src/compiler/compile/render_dom/wrappers/Window.ts

```diff
@@ -52,12 +52,15 @@ export default class WindowWrapper extends Wrapper {
 		add_event_handlers(block, '@_window', this.handlers);
 
 		this.node.bindings.forEach(binding => {
+			// Todo: what if it's a MemberExpression?
```

Comment:
I normally see `TODO` in all caps. Might be good to keep it consistent in case someone greps for open todos

```suggestion
			// TODO: what if it's a MemberExpression?
```